runbook.md

```diff
@@ -69,6 +69,12 @@ As an example an incident in February 2020 occurred because an `FTB Article` ass
 
 If _nobody_ can see their icons, then this is a more serious problem and should be pushed to Second Line.
 
+#### __Salesforce usage__
+The app connects to Salesforce to get contract details for the user, and updates the Syndication database when it receives them
+* User: `next-syndication`
+* URLs: Logs in and uses an SDK, retrieves details from `/SCRMContract/[someContractID]`
+* Splunk: [index="heroku" source=\*syndication-api\* salesforce error| spath "message.error message" | search "message.error message"=NullApexResponse](https://financialtimes.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22heroku%22%20source%3D*syndication-api*%20salesforce%20error%7C%20spath%20%22message.error%20message%22%20%7C%20search%20%22message.error%20message%22%3DNullApexResponse&sid=1661942615.1390859&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=-1h&latest=now)
```

Comment:
**Comment**: Good call on including links for the splunk queries!
Please check this URL: https://financialtimes.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22heroku%22%20source%3D%5C*syndication-api%5C*%20salesforce%20error%7C%20spath%20%22message.error%20message%22%20%7C%20search%20%22message.error%20message%22%3DNullApexResponse&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=-1w&latest=now&sid=1661945757.1404098
you can compare it with this one: https://financialtimes.splunkcloud.com/en-US/app/search/search?q=search%20index%3D%22heroku%22%20source%3D%5C*syndication-api%5C*%20salesforce%20error&display.page.search.mode=smart&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=-1w&latest=now&sid=1661945817.1404271
I think `index="heroku" source=\*syndication-api\* salesforce error` is simpler than `index="heroku" source=\*syndication-api\* salesforce error| spath "message.error message" | search "message.error message"=NullApexResponse` to help drill down after.
I would even prefer `index="heroku" source=*syndication* salesforce error` but that's a matter of taste.
lmk your thoughts

Reply:
something's not working anymore with the first url I put in, I think some underscores got removed, so I'll fix that.

I'd picked the NullApexResponse error to limit it down to just the failing salesforce calls, but the reduced filters will show more of the 'surrounding' errors that give more details like what the licence ID was.